docker/build.sh

```diff
@@ -85,6 +86,16 @@ if [[ "$1" == "--cache-from" ]]; then
     shift 1
 fi
 
+if [[ "$1" == "--context-path" ]]; then
+    shift 1
+    DOCKER_CONTEXT_PATH="$1"
```

Comment:
To be consistent with the rest of this file, we should use `$2` here and `shift 2` lower down.